frontend/src/components/Selectors/StateSelector.tsx

```diff
@@ -21,7 +22,15 @@ export let StateSelector: React.StatelessComponent<StateSelectorProps> = ({ clas
     )}
   </DropDownMenu >
 
+/**
+ * As part of State selection, send an event to GA.
```

Comment:
I meant docstring explaining which State selector this is :)

Reply:
Done!